wsmaster/che-core-api-metrics/src/main/java/org/eclipse/che/api/metrics/WorkspaceStartAttemptsMeterBinder.java

```diff
@@ -40,13 +43,22 @@ public void bindTo(MeterRegistry registry) {
         Counter.builder(workspaceMetric("starting_attempts.total"))
             .description("The count of workspaces start attempts")
             .register(registry);
+    startingDebugCounter =
+        Counter.builder(workspaceMetric("starting_attempts.total"))
+            .tags(withStandardTags("debug"))
```

Comment:
Tags are key-value pairs... Though I must agree this is not particularly well explained in the method signature or javadoc.

Reply:
fixed